internal/transport/grpc/server/users.go

```diff
@@ -0,0 +1,36 @@
+package server
+
+import (
+	"context"
+	pb2 "workshop/internal/transport/grpc/pb"
+	"workshop/internal/transport/http/handlers"
+	"workshop/internal/users"
+)
+
+type Users struct {
+	pb2.UnimplementedUsersServer
+	user handlers.UsersService
+	repo users.Repository
+}
+
+func NewUsers(us handlers.UsersService, repo users.Repository) *Users {
+	return &Users{user: us, repo: repo}
+}
+
+func (u *Users) Create(ctx context.Context, in *pb2.CreateUserRequest) (*pb2.User, error) {
+	user, err := u.user.Create(ctx, in.GetName())
+	if err != nil {
+		return nil, err
```

Comment:
as well as HTTP, GRPC provides a response code. Try this https://pkg.go.dev/google.golang.org/grpc/status#Error

Reply:
done